### ISSUE TITLE
mzXML Validation

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#2556]
Does not work anymore. On nightly linux and nightly MacOS.
```
checking [EXTRA] static bool isValid(const String& filename) ...
    creating new temporary filename 'MzXMLFile_test_564.tmp' (line 564)
Validation error in file 'MzXMLFile_test_564.tmp' line 0 column 0: unsupported protocol in URL
Validation error in file 'MzXMLFile_test_564.tmp' line 4 column 88: fatal error during schema scan
Validation error in file 'MzXMLFile_test_564.tmp' line 0 column 0: unsupported protocol in URL
Validation error in file 'MzXMLFile_test_564.tmp' line 4 column 155: fatal error during schema scan
 -  line 567:  TEST_EQUAL(f.isValid(tmp_filename, std::cerr),true): got 0, expected 1
: failed
```
It changed with @cbielows changes to the schema but validates fine on Travis, on @cbielows PC and online.
Possible errors:
1) Xerces version
2) Xerces double linking (I tried the same fix as for the FeatureFinder; did not help)
3) Xerces parser incorrect deallocation
- What happens if there is an Exception?
https://github.com/OpenMS/OpenMS/blob/36ccb5c6d443b4ec59b18a6c1cc315e29b516b9c/src/openms/source/FORMAT/VALIDATORS/XMLValidator.cpp#L99
- I read somewhere that initialize should always go together with terminate
https://github.com/OpenMS/OpenMS/blob/36ccb5c6d443b4ec59b18a6c1cc315e29b516b9c/src/openms/source/FORMAT/VALIDATORS/XMLValidator.cpp#L69
3) Namespace location lookup (I tried several variations there already)

**Comment by jpfeuffer** [comment]
OR: xerces was built with no or only basic netAccessor support. I will check.
http://xerces.apache.org/xerces-c/build-3.html

**Comment by jpfeuffer** [comment]
Yes, by default on mac and linux we build xerces in the contrib with --disable-network.
Why?

**Comment by jpfeuffer** [comment]
The old schema had no online includes, I think.

**Comment by cbielow** [comment]
then I guess removing the flag is the best option?!
I guess if someone wants to run `FileInfo -v ...` on a HPC cluster without internet access, thats a problem, but we'd have to merge all dependencies in our xml schemata in order to avoid this... this seems wrong.

**Comment by hroest** [comment]
> What happens if there is an Exception?

we should fix that but its not the most important thing e.g. few people will ever run the validator. otherwise we should copy paste code from here: https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/FORMAT/XMLFile.cpp#L164

**Comment by jpfeuffer** [comment]
@timosachsenberg This issue causes the mzXML tests to fail.

**Comment by jpfeuffer** [comment]
solved with #2715
